### DISCUSSION TITLE
Pill - Change icon with "removeButtonProps" type issue

### DISCUSSION THREAD

**Discussion by realyuki** [mantinedev/mantine#5998] [Q&A]
hi. I have some issue to change remove button icon.
there is no CloseButton types in removeButtonProps. Is there any other way to change the button icon?

```ts
<MantinePill
  size="xs"
  radius="xs"
  removeButtonProps={
    {
      icon: <Icon type="clear" size={24} /> // type error
    }
  }
/>
```

**Comment by rtivital** [comment]
It will be fixed in one of the next patches
